internal/protocol/connector.go

```diff
@@ -53,6 +53,10 @@ func NewConnector(id uint64, store NodeStore, config Config, log logging.Func) *
 		config.BackoffCap = time.Second
 	}
 
+	if config.ConcurrentLeaderConns == 0 {
+		config.ConcurrentLeaderConns = 10
+	}
```

Comment:
Can this be avoided by having `Driver.concurrentLeaderConns` default to MaxConcurrentLeaderConns?

Reply:
Yes, though I had to move the constant to the `protocol` package to avoid an import cycle.